pkg/linode-bs/nodeserver.go

```diff
@@ -261,8 +298,8 @@ func (ns *LinodeNodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeSt
 		}
 		options = append(options, mnt.MountFlags...)
 	} else if blk := volumeCapability.GetBlock(); blk != nil {
-		// TODO(#64): Block volume support
-		return nil, status.Error(codes.Unimplemented, "Block volume support is not yet implemented")
+		// If the access type is block, do nothing for stage
+		return &csi.NodeStageVolumeResponse{}, nil
```

Comment:
since this does nothing now maybe move it up to line 290 in its own if check like the `!notMnt` does? seems messy for an else if just for this now

Reply:
Kk